NewsScraper/DawnCrawler/DawnSpiders/items.py

```diff
@@ -0,0 +1,20 @@
+import logging
+
+import scrapy
+from scrapy.utils.log import configure_logging
+
+
+class ArticleItem(scrapy.Item):
+    configure_logging(install_root_handler=False)
```

Comment:
Why not use `LOG_FILE` setting ? [See here](https://doc.scrapy.org/en/latest/topics/settings.html#log-file).